package.json

```diff
@@ -40,7 +39,7 @@
     "@vercel/analytics": "^1.3.1",
     "@vercel/blob": "^0.24.1",
     "@vercel/postgres": "^0.10.0",
-    "ai": "4.2.10",
+    "ai": "4.3.4",
```

Comment:
perhaps worth `pnpm update --latest '@ai-sdk/*' ai` else we are flying a little blind re: compatibility of the new with the old